sensor_msgs/PointCloud2 kinectInputPt
sensor_msgs/PointCloud2 maskInputPt 
---
sensor_msgs/PointCloud2 outputPt